platform/src/main/java/org/stellar/anchor/platform/AnchorPlatformServer.java

```diff
@@ -31,6 +31,7 @@ public static ConfigurableApplicationContext start(
                 //       disableMetrics param -
                 //  https://github.com/stellar/java-stellar-anchor-sdk/issues/297
                 "spring.mvc.converters.preferred-json-mapper=gson",
+                "spring.config.import=optional:classpath:example.env[.properties]",
```

Comment:
❓ What is the purpose of this line here?

IMHO, we should avoid using the example config directly. I also plan to refactor this part of the code to avoid setting `spring` configurations directly in the `AnchorPlatforrmServer`.

Reply:
It'd be good to have a way for users to run the Platform on localhost with little to no friction though. I'll put some thought into it 🤔 .